auth-web/src/views/auth/staff/StaffDashboardView.vue

```diff
@@ -287,24 +324,27 @@ export default defineComponent({
     ]
 
     const localVars = (reactive({
-      businessIdentifier: '',
-      searchedBusinessNumber: '',
-      searchActive: false,
-      errorMessage: '',
-      canViewIncorporationSearchResult: false,
       affiliatedOrg: {},
+      businessIdentifier: '',
       canSearchFAS: computed((): boolean => currentUser.value?.roles?.includes(Role.FasSearch)),
       canViewAccounts: computed((): boolean => currentUser.value?.roles?.includes(Role.StaffViewAccounts)),
       canViewAllTransactions: computed((): boolean => currentUser.value?.roles?.includes(Role.ViewAllTransactions)),
-      canViewGLCodes: computed((): boolean => currentUser.value?.roles?.includes(Role.ManageGlCodes)),
       canViewEFTPayments: computed((): boolean => currentUser.value?.roles?.includes(Role.ManageEft)),
+      canViewGLCodes: computed((): boolean => currentUser.value?.roles?.includes(Role.ManageGlCodes)),
+      canViewIncorporationSearchResult: false,
+      errorMessage: '',
       isFasDashboardEnabled: computed((): boolean => currentUser.value?.roles?.includes(Role.FasSearch)),
+      registrySearchUrl: computed((): string => ConfigHelper.getRegistrySearchUrl()),
+      searchActive: false,
+      searchedBusinessNumber: '',
       showBusSearchlink: computed((): boolean => true),
-      registrySearchUrl: computed((): string => ConfigHelper.getRegistrySearchUrl())
+      showInvoluntaryDissolutionTile: computed((): boolean => LaunchDarklyService.getFlag(LDFlags.EnableInvoluntaryDissolution) || false)
```

Comment:
I just alphabetized these and added this here. This is the computed value of the involuntary dissolution FF.